app/models/patient.rb

```diff
@@ -162,6 +164,33 @@ def event_params
     }
   end
 
+  def confirm_unique_phone_number
+    ##
+    # This method is preferred over Rail's built-in uniqueness validator
+    # so that case managers get a meaningful error message when a patient
+    # exists on a different line than the one the volunteer is serving.
+    #
+    # See https://github.com/DCAFEngineering/dcaf_case_management/issues/825
+    ##
+    # phones are unique, so there should be zero or one match
+    match = Patient.where(primary_phone: primary_phone).first
+
+    if match
+      # skip when an existing patient updates and matches itself
+      if match._id == self._id
```

Comment:
Ah, I see what you mean here. You're right that the underscore is a mongoid internal thing and it's best not to use it. Fortunately, IIRC Mongoid also provides a regular `id` that you can use, I think. That said, see above for a suggestion that might make this a little straightforward?